buildSrc/build.gradle.kts

```diff
@@ -39,12 +39,12 @@ repositories {
 dependencies {
     implementation(files(libs.javaClass.superclass.protectionDomain.codeSource.location))
 
-    implementation(libs.detektPlugin)
-    implementation(libs.dokkatooPlugin)
-    implementation(libs.graalVmNativeImagePlugin)
     implementation(libs.jgit)
-    implementation(libs.kotlinPlugin)
-    implementation(libs.mavenPublishPlugin)
+    implementation(libs.plugin.detekt)
```

Comment:
@mnonnenmacher: I believe it was an oversight of us not to name this plural "plugin*s*". I'll amend your PR.

Reply:
Ok, I just realized that's not possible due to a name clash with the `[plugins]` section, reverted.